MEMBER_TRAVEL_FUND.md

```diff
@@ -6,12 +6,14 @@ and the Node.js Foundation projects.
 
 ## Restrictions
 
-* Candidates must be an individual member of the Node.js Foundation.
+* Candidates must be an individual member of the Node.js Foundation. The membership
+  status can be checked on [the foundation website][].
+* Requests must include a note that their employer cannot fund that request.
```

Comment:
Total nit, but it might be nice to word this in a way that people don't feel compelled to reveal that they are unemployed (students, between jobs, whatever), and also in a way that covers confirming that the event won't cover the costs either. Will need some word-smithing, and can totally happen in a subsequent PR anyway, but maybe something like this?:

```markdown
* Requester must indicate that they do not have funding available from another source,
  such as an employer willing to cover the costs or the event covering costs for presenters or others.
```